packages/data-stores/src/plans/test-utils/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import type { PlanFeature } from '../types';
+
+export const buildPlanFeaturesDict = (
```

Comment:
❤️  this method and I hope that we can extend our testing utils in the future to make generating mocks easier!